__tests__/database/migration/index.test.ts

```diff
@@ -5,7 +5,7 @@ describe('TESTING migration file', () => {
     expect(DROP_ALL_DB).toBeFalsy();
   });
 
-  it('TEST should expect TARGET_MIGRATION_VERSION to be 7', () => {
-    expect(TARGET_MIGRATION_VERSION).toBe(7);
+  it('TEST should expect TARGET_MIGRATION_VERSION to be 8', () => {
+    expect(TARGET_MIGRATION_VERSION).toBe(8);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case is updated correctly to reflect the new `TARGET_MIGRATION_VERSION` value. However, it's important to ensure that the migration scripts are also updated accordingly to handle this new version.
```diff
-    expect(TARGET_MIGRATION_VERSION).toBe(7);
+    expect(TARGET_MIGRATION_VERSION).toBe(8);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->